runtime/mainnet/src/config/system.rs

```diff
@@ -0,0 +1,490 @@
+			assert_eq!(actual.base_extrinsic, expected.base_extrinsic);
+			assert_eq!(actual.max_extrinsic, expected.max_extrinsic);
+			assert_eq!(actual.max_total, expected.max_total);
+			assert_eq!(actual.reserved, expected.reserved);
+		}
+	}
+
+	#[test]
+	fn system_db_weight_uses_rocks_db() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::DbWeight>(),
+			TypeId::of::<RocksDbWeight>(),
+		);
+	}
+
+	#[test]
+	fn system_uses_blake2_hashing() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::Hashing>(),
+			TypeId::of::<BlakeTwo256>(),
+		);
+	}
+
+	#[test]
+	fn system_uses_multi_address_lookup() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::Lookup>(),
+			TypeId::of::<AccountIdLookup<AccountId, ()>>(),
+		);
+	}
+
+	#[test]
+	fn system_max_consumers_limited_to_16() {
+		assert_eq!(<<Runtime as frame_system::Config>::MaxConsumers as Get<u32>>::get(), 16);
+	}
+
+	#[test]
+	fn system_multi_block_migrator_disabled() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::MultiBlockMigrator>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn system_nonce_uses_u32() {
+		assert_eq!(TypeId::of::<<Runtime as frame_system::Config>::Nonce>(), TypeId::of::<u32>(),);
+	}
+
+	#[test]
+	fn system_account_killed_handler_disabled() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::OnKilledAccount>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn system_new_account_handler_disabled() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::OnNewAccount>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn system_set_code_handler_managed_by_parachain_system() {
+		// Runtime upgrades orchestrated by parachain system pallet
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::OnSetCode>(),
+			TypeId::of::<cumulus_pallet_parachain_system::ParachainSetCode<Runtime>>(),
+		);
+	}
+
+	#[test]
+	fn system_post_inherent_handler_disabled() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::PostInherents>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn system_post_transactions_handler_disabled() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::PostTransactions>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn system_pre_inherent_handler_disabled() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::PreInherents>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn system_single_block_migrations_is_empty() {
+		assert_eq!(
+			TypeId::of::<<Runtime as frame_system::Config>::SingleBlockMigrations>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	fn system_ss58_prefix_matches_relay() {
+		assert_eq!(<<Runtime as frame_system::Config>::SS58Prefix>::get(), 0);
+	}
+
+	#[test]
+	fn system_does_not_use_default_weights() {
+		assert_ne!(
+			TypeId::of::<<Runtime as frame_system::Config>::SystemWeightInfo>(),
+			TypeId::of::<()>(),
+		);
+	}
+
+	#[test]
+	#[ignore]
```

Comment:
Remove ignore after benchmarks results are included in the configuration of frame_system.